### PR TITLE
[html-pdf] Add localUrlAccess added in v3

### PR DESCRIPTION
**Pull request by zbarbuto**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [See the v3 changelog here](https://github.com/marcbachmann/node-html-pdf/releases)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/html-pdf/index.d.ts [hunk 1]
```diff
@@ -57,6 +57,13 @@ declare module 'html-pdf' {
     // Script options
     phantomPath?: string | undefined;
     phantomArgs?: string[] | undefined;
+
+    // Prevent local file:// access by passing '--local-url-access=false' to phantomjs
+    // For security reasons you should keep the default value if you render
+    // arbitrary html/js.
+    // The default is `false`
+    localUrlAccess: boolean | undefined;
```
Comment (westy92) [inline types/html-pdf/index.d.ts:65]:
```suggestion
    localUrlAccess?: boolean | undefined;
```

**Review by westy92 — CHANGES_REQUESTED** [review]
Looks good, but I think the option should be optional.

File: types/html-pdf/index.d.ts (hunk 1 above)
Comment (zbarbuto) [reply types/html-pdf/index.d.ts]: Thanks, done.

**Review by westy92 — APPROVED** [review]

**Comment by zbarbuto** [comment]
Ready to merge

**Pull request merged**